frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditDate/EditDate.tsx

```diff
@@ -54,9 +84,24 @@ const transformDateFieldToEditForm = (field: DateFieldBase): EditDateInputs => {
       ? transformDateToShortIsoString(field.dateValidation.customMinDate) ?? ''
       : ('' as const),
   }
+
+  const nextAddParticularDayRestriction = !!field?.invalidDays?.length
```

Comment:
will this cause the toggle to turn off when user unchecks all options in the UI? is that intended behaviour?

Reply:
If user unchecks all options, the `invalidDays` array will be of length > 0 which will assign `true` to `nextAddParticularDayRestriction` and the toggle will be turned on still. However if the user checks all the options, the toggle will still be turned on. I checked with Pearly, if this is the supposed intended behaviour and she said she will check with Shanty and get back to me.